hbase-server/src/main/java/org/apache/hadoop/hbase/master/ServerManager.java

```diff
@@ -78,13 +75,18 @@
 import org.apache.hadoop.hbase.regionserver.RegionOpeningState;
 import org.apache.hadoop.hbase.security.User;
 import org.apache.hadoop.hbase.util.Bytes;
+import org.apache.hadoop.hbase.util.EnvironmentEdgeManager;
 import org.apache.hadoop.hbase.util.RetryCounter;
 import org.apache.hadoop.hbase.util.RetryCounterFactory;
 import org.apache.hadoop.hbase.util.Triple;
 import org.apache.hadoop.hbase.zookeeper.ZKUtil;
 import org.apache.hadoop.hbase.zookeeper.ZooKeeperWatcher;
 import org.apache.zookeeper.KeeperException;
 
+import com.google.common.annotations.VisibleForTesting;
+import com.google.protobuf.ByteString;
+import com.google.protobuf.ServiceException;
+
```

Comment:
@sguggilam looks like import order is incorrect as per checkstyle error:
```
ServerManager.java:86:import com.google.common.annotations.VisibleForTesting;: Wrong order for 'com.google.common.annotations.VisibleForTesting' import. [ImportOrder]
```
Maybe you can put these import in their original place, (could be IDE settings issue)

Reply:
It was using default alphabetical sorting order....imported the hbase import file in eclipse now...fixed and pushed the change